.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -148,9 +148,11 @@ jobs:
   # Fail if the triggering actor is not part of Zama organization.
```

Comment:
thinking about the aggregate file changes above, there is no outputs.any_changed that is global to avoid doing it by hand ?